docs/tutorials/get-started-kustomize.md

```diff
@@ -24,26 +24,11 @@ First you'll need a git repository to store your cluster desired state.
 In our example we are going to use [`fluxcd/flux-get-started`](https://github.com/fluxcd/flux-get-started).
 If you want to use that too, be sure to create a fork of it on GitHub.
 
-Create a directory and add the `flux` namespace definition to it:
-
-```sh
-mkdir fluxcd
-
-cat > fluxcd/namespace.yaml <<EOF
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: flux
-EOF
-```
-
-Create a kustomization file and use the Flux deploy YAMLs as base:
+Create a directory, and add a `kustomization.yaml` file that uses the
+Flux deploy YAMLs as a base:
 
 ```sh
 cat > fluxcd/kustomization.yaml <<EOF
-namespace: flux
```

Comment:
I would keep the namespace declaration to make it obvious that Flux will be deployed in `flux` namespace.